Sources/SwiftParser/Types.swift

```diff
@@ -687,27 +687,74 @@ extension Parser.Lookahead {
     return true
   }
 
-  mutating func skipTypeAttributeList() {
+  mutating func skipTypeAttributeList() -> Bool {
```

Comment:
Could you add a short comment to indicate what the bool return value means?

Reply:
I think we can probably just rename this to `canParseTypeAttributeList` which would then make it obvious?

Reply:
Yeah, I like `canParseTypeAttributeList`, renamed!